app/controllers/articles_controller.rb

```diff
@@ -254,7 +254,7 @@ def article_params_json
     if params["article"]["series"].present?
       collection = Collection.find_series(params["article"]["series"], @user)
       params["article"]["collection_id"] = collection.id
-    elsif params["article"]["series"] == "" # reset collection?
+    else # reset collection?
```

Comment:
This a logic change. Having an `elsif` means that this whole construct could be a no-op, which is not true with an `else` since there will always be one branch executing. I don't feel particularly good with this change unless we make sure there's excellent test coverage around this.

Reply:
It really isn't the same. I'll remove this change and open another PR to treat the exceeded lines.